### PR TITLE
Build out action rows on dashboard

### PR DESCRIPTION
**Pull request by juliendonck**
Starting to build out these actionrows:

Screenshot 2022-05-06 at 13 01 49

### CODE REVIEW

File: src/lib/components/WorkstreamCard/Actions.svelte [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  let creator: boolean =
+    $connectedAndLoggedIn && workstream.creator === $walletStore.accounts[0];
+  let assignee: boolean =
+    $connectedAndLoggedIn &&
+    workstream.acceptedApplication === $walletStore.accounts[0];
+  let applicant: boolean =
+    $connectedAndLoggedIn &&
+    workstream.applicants &&
+    workstream.applicants.includes($walletStore.accounts[0]);
```
Comment (efstajas) [inline src/lib/components/WorkstreamCard/Actions.svelte:21]:
```suggestion
    workstream.applicants?.includes($walletStore.accounts[0]);
```

File: src/lib/components/WorkstreamCard/Actions.svelte [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  let rejectant: boolean =
+    $connectedAndLoggedIn &&
+    workstream.rejectedApplications &&
+    workstream.rejectedApplications.includes($walletStore.accounts[0]);
```
Comment (efstajas) [inline src/lib/components/WorkstreamCard/Actions.svelte:25]:
```suggestion
    workstream.rejectedApplications?.includes($walletStore.accounts[0]);
```

File: src/lib/stores/workstreams/types.ts [hunk 3]
```diff
@@ -30,6 +30,7 @@ export interface Workstream {
   total: Money;
   title: string;
   desc: string;
+  assignee?: string;
```
Comment (efstajas) [inline src/lib/stores/workstreams/types.ts:33]: This probably came from a bad merge?

```suggestion
```

**Review by efstajas — CHANGES_REQUESTED** [review]
Looks great and works great, just a few small notes ✨

File: src/lib/stores/workstreams/types.ts (hunk 3 above)
Comment (juliendonck) [reply src/lib/stores/workstreams/types.ts:33]: No I added that one, did we remove that?

**Review by efstajas — APPROVED** [review]
👍

**Pull request merged**